provision/provision.go

```diff
@@ -702,6 +703,23 @@ type TsuruYamlHealthcheck struct {
 	TimeoutSeconds  int    `json:"timeout_seconds" yaml:"timeout_seconds" bson:"timeout_seconds,omitempty"`
 }
 
+type TsuruYamlKubernetesConfig struct {
+	Groups map[string]TsuruYamlKubernetesGroup `bson:",omitempty"`
```

Comment:
This will be a pain to handle... Both pod names and process names can have `.` in their names. Serializing this to mongodb as a map this will cause an error as keys aren't allowed to have dots. We would need to change the way we serialize this to mongodb to use a list with keys and values to correctly allow dots here. This will probably mean using different structs for yaml parsing and database serialization. :(